modules/exploits/windows/http/tomcat_cgi_cmdlineargs.rb

```diff
@@ -118,16 +114,6 @@ def on_new_session(cli)
   end
 
   def exploit
-    print_status("Checking if #{rhost} is vulnerable")
-    unless check == CheckCode::Vulnerable
-      unless datastore['ForceExploit']
-        fail_with(Failure::NotVulnerable, 'Target is not vulnerable. Set ForceExploit to override.')
-      end
-
-      print_warning('Target does not appear to be vulnerable.')
-    end
-
-    print_status("#{rhost} seems vulnerable, what a good day.")
```

Comment:
I didn't initially intend to pull this functionality into the AutoCheck mixin - but it looks like it will be useful to help remove boilerplate 👀

Reply:
If we're happy with the approach of the PR it should be possible to roll this out to the other modules too

Reply:
![image](https://user-images.githubusercontent.com/4551878/86143727-69353700-baba-11ea-9094-90f5fd918464.png)

Sounds good to me.

Reply:
Let's roll it in another PR?